pkgs/by-name/mo/moneydance/package.nix

```diff
@@ -53,7 +53,11 @@ stdenv.mkDerivation (finalAttrs: {
     description = "An easy to use and full-featured personal finance app that doesn't compromise your privacy";
     sourceProvenance = [ lib.sourceTypes.binaryBytecode ];
     license = lib.licenses.unfree;
-    platforms = jdk.meta.platforms;
+    # Darwin refers to Zulu Java, which breaks the evaluation of this derivation
+    # for some reason
```

Comment:
zulu java doesn't seem have an argument named `openjfx`, which seems to be causing the error.
```
 error: function 'anonymous lambda' called with unexpected argument 'openjfx'

       at /Users/anna/src/nix.1/pkgs/development/compilers/zulu/common.nix:1:1:
```